common/src/main/java/org/apache/cassandra/sidecar/common/exceptions/NodeBootstrappingException.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.cassandra.sidecar.common.exceptions;
+
+/**
+ * Exception thrown when a node is bootstrapping while an operation cannot be performed when the node is bootstrapping
+ */
+public class NodeBootstrappingException extends RuntimeException
```

Comment:
How about `IllegalCassandraStateException`? The exception is thrown when then state of Cassandra is not expected. Regarding node vs cassandra, I'd prefer using "Cassandra", although at many places both are used interchangeably. 
My understanding of `NodeBootstrappingException` is that there is some exception during node bootstrapping. It confused me at the beginning.